src/test/java/bio/terra/service/journal/flight/CreateProfileJournalEntryStepTest.java

```diff
@@ -0,0 +1,67 @@
+package bio.terra.service.journal.flight;
+
+import static bio.terra.service.common.CommonFlightUtils.getFlightInformationOfInterest;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import bio.terra.common.category.Unit;
+import bio.terra.common.fixtures.AuthenticationFixtures;
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import bio.terra.model.BillingProfileRequestModel;
+import bio.terra.service.auth.iam.IamResourceType;
+import bio.terra.service.journal.JournalService;
+import bio.terra.service.profile.flight.create.CreateProfileJournalEntryStep;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.StepStatus;
+import java.util.UUID;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.test.context.ActiveProfiles;
+
+@RunWith(MockitoJUnitRunner.StrictStubs.class)
+@ActiveProfiles({"google", "unittest"})
+@Category(Unit.class)
```

Comment:
It would be nice to create these tests using JUnit 5, after updating your branch from `develop`. 
```suggestion
@ExtendWith(MockitoExtension.class)
@Tag("bio.terra.common.category.Unit")
```
If this test doesn't inject any application context beans, then I don't think it needs any `ActiveProfiles`, so that can be removed.

I think the only other changes needed are to change `@Before` to `@BeforeEach` and change the import of `Test` to be `org.junit.jupiter.api.Test`

Reply:
Also in junit 5, you no longer need to declare the class or test methods `public`. So you can omit the `public` modifiers here too.